runtime/POSIX/input_output.c

```diff
@@ -1,7 +1,82 @@
 #include <stdio.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <errno.h>
+
+/*
+ * Return the (stdio) flags for a given mode. Store the flags
+ * to be passed to an open() syscall through *optr.
+ * Return 0 on error.
+ */
+int count_flags(const char *mode, int *optr) {
```

Comment:
Bad naming. Nothing is counting, but `convert_to_stdio_open_flags`.
For such a namning you don't need a function doc)

Reply:
Change